geaflow/geaflow-core/geaflow-api/src/main/java/com/antgroup/geaflow/env/IEnvironment.java

```diff
@@ -42,9 +42,9 @@ public interface IEnvironment extends Serializable {
     enum EnvType {
 
         /**
-         * Ray cluster.
+         * Community ray cluster.
          */
-        RAY,
+        RAY_COMMUNITY,
```

Comment:
I'm concerning about this. Why we rename ray to ray community? For the open source software, the meaning of "ray" is "open source ray" by default. I think we don't need to rename it because there is only one ray project in community currently. "RAY_COMMUNITY" is confusing for open source users.